src/hooks/JB721Delegate/transactor/ReconfigureV2V3FundingCycleWithNftsTx.ts

```diff
@@ -57,7 +58,8 @@ export function useReconfigureV2V3FundingCycleWithNftsTx(): TransactorInstance<R
     const JBTiered721DelegateStoreAddress =
       await findJBTiered721DelegateStoreAddress()
 
-    const collectionName = collectionMetadata.name
+    const collectionName =
```

Comment:
**question** how is it possible to get to this stpe iwthout setting a name?

Reply:
Yep should be a compulsory field on the add NFTs form. I've done so now (`AddNftsSection:35`)